addon/addon.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: kubermatic.k8s.io/v1
```

Comment:
Maybe this file should rather go int the examples/ folder? As is is not part of the addon itself.

Reply:
yep, you are right, will move it

Reply:
Done, also I have tested with dev cluster, and it works.